prac_05/state_names.py

```diff
@@ -0,0 +1,18 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD":"Queensland", "NSW": "New South Wales", "NT" : "Northern Territory", "WA" : "Western Australia",
+            "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ")
+while state_code != "":
```

Comment:
I think it might be the wrong commit, I did commit the file then do some updates and committed again under "Update Australian state names program" some of the review messages are also saying outdated at the top, and some aren't. I'm not too sure, I'll check with lindsay on Wednesday. Thanks